docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Audit logs
+description: A list of changes that happened to selected entities owned by an account.
+---
+
+## What are audit logs?
+
+Our implementation of audit logs will record actions made with Expo services by accounts. Recorded data will include what object was affected, what type of modification was made, who performed the action, and when this activity occurred.
+
+Because audit logs can only be created and never modified or deleted, they will serve as a source of truth to help monitor events and possibly debug issues occurring within accounts and products.
+
+Audit logs are available for enterprise plan customers only. Immediately after subscribing, some of the logs will become available immediately as they could have been used internally by Expo. Some other types of logs will start to be collected after the subscription is activated.
+
+They can be accessed for any account under [expo.dev/accounts/[account]/settings/audit-logs](https://expo.dev/accounts/[account]/settings/audit-logs).
+
+## Audit log use cases
+
+### Permission monitoring
+
+Audit logs can be used to track user invitations and permission changes within your organization. An example security event could include an employee account being compromised and used to invite an attacker into an organization and change their permission to admin.
+
+In this scenario, audit logs would record which employee account invited an account and modified permissions, and since audit logs are immutable, the attacker would not be able to delete this recorded history. Other organization members would be able to review the audit logs to determine which account was compromised and take action to revoke the attacker's permissions and secure the employee's account.
```

Comment:
```suggestion
In this scenario, audit logs would record which employee account invited the attacker and modified permissions. Since audit logs are immutable, the attacker would not be able to delete this recorded history. Other organization members will be able to review the audit logs to determine which account was compromised, take action to revoke the attacker's permissions and secure the employee's account.
```